glassfish-ejb-jar_3_1-1.dtd: Element type property declared more than once

# Description #
----------
I think [glassfish-ejb-jar_3_1-1.dtd](https://github.com/payara/Payara/blob/master/appserver/deployment/dtds/src/main/resources/glassfish/lib/dtds/glassfish-ejb-jar_3_1-1.dtd) violates  [Validity Constraint: Unique Element Type Declaration](https://www.xml.com/axml/target.html#AX-ElDecOnce).

## Expected Outcome

`glassfish-ejb-jar_3_1-1.dtd` distributed with Payara to be usable in validation of `glassfish-ejb-jar.xml`s, as [the one from Glassfish is](https://travis-ci.org/pzrep/Payara/jobs/546451568) (using [The XML Maven Plugin](http://www.mojohaus.org/xml-maven-plugin/index.html)).

## Current Outcome
Problem with validation of `glassfish-ejb-jar.xml` against this DTD, due to: [Element type "property" must not be declared more than once](https://travis-ci.org/pzrep/Payara/jobs/557768164#L462).

## Steps to reproduce (Only for bug reports) 
Validate [any] `glassfish-ejb-jar.xml` against dtd from Payara distribution
- https://travis-ci.org/pzrep/Payara/jobs/557768164 to get:

```
[ERROR] Failed to execute goal org.codehaus.mojo:xml-maven-plugin:1.0.2:validate (default) on project dtd-issue: While parsing /home/travis/build/pzrep/Payara/dtd-issue/src/main/resources/glassfish-ejb-jar.xml, at Public ID -//GlassFish.org//DTD GlassFish Application Server 3.1 EJB 3.1//EN, file:/home/travis/build/pzrep/Payara/appserver/deployment/dtds/src/main/resources/glassfish/lib/dtds/glassfish-ejb-jar_3_1-1.dtd, line 1224,  column 35: error: Element type "property" must not be declared more than once.
```
or
- more forgiving - `xmlstarlet validate --err --dtd glassfish-ejb-jar_3_1-1.dtd glassfish-ejb-jar.xml` to get:
```
glassfish-ejb-jar_3_1-1.dtd:1224.35: Redefinition of element property
<!ELEMENT property (description?)>
                                  ^
glassfish-ejb-jar.xml - valid
```

### Samples

https://github.com/pzrep/Payara/tree/bad_dtd

## Context (Optional)

<!-- Give details on this issue has affected you, for example: What are you trying to accomplish?
Providing context helps us come up with a solution that is most useful for your scenario. -->

I'm trying to validate EJB descriptor before deployment.

## Environment ##

- **Payara Version**: Payara Server  5.192 #badassfish (build 115)
- **Edition**: Full
- **JDK Version**: openjdk version "1.8.0_212"
- **Operating System**: Linux
- **Database**: N/A

--- a/appserver/deployment/dtds/src/main/resources/glassfish/lib/dtds/glassfish-ejb-jar_3_1-1.dtd
+++ b/appserver/deployment/dtds/src/main/resources/glassfish/lib/dtds/glassfish-ejb-jar_3_1-1.dtd
@@ -701,11 +701,6 @@ for delivering messages to the message-driven bean.
 -->
 <!ELEMENT resource-adapter-mid ( #PCDATA ) >
 
-<!-- 
-Generic name-value pairs property
--->
-<!ELEMENT property ( name, value ) >
-
 <!--
 This text nodes holds a value string.
 -->
@@ -1221,9 +1216,9 @@ The version-identifier element contains the version information. The value is
 <!-- 
 Syntax for supplying properties as name value pairs 
 -->
-<!ELEMENT property (description?)>
-<!ATTLIST property name  CDATA  #REQUIRED
-                   value CDATA  #REQUIRED>
+<!ELEMENT property (description?, name?, value?)>
+<!ATTLIST property name  CDATA  #IMPLIED
+                   value CDATA  #IMPLIED>
 
 <!--
 Optional Default authentication configuration for an EJB web service endpoint.

